docs/6.-Engineering/Source-Control,-Versioning-&-Branching-Strategy/README.md

```diff
@@ -25,9 +25,10 @@ Code is managed and stored using [Git](https://git-scm.com/docs) and GitHub.
 | **Branch** | **Key Purpose** | **Naming** |
 |--|--|--|
 | **Main** | Reflection of production environments. At any point you should be able to branch from Main absolutely confident that it will match what is  | `main` |
-| **Release** | A release branch is used when EITHER of following conditions are met: <br> 1. multiple features are being released together <br> 2. BAT is required. | `release/{Release-Name]` |
-| **Feature** | Contains all the changes required for a specific feature or bug, only. Note that the term "feature branch is used as it is common practice, but it is synonymous with a backlog item - an independent, releasable, valuable product increment. | `feature/{Feature-Name}` <br> OR <br> `bug/{Bug-Name}` |
+| **Release** | A release branch is used when EITHER of following conditions are met: <br> 1. multiple features are being released together <br> 2. BAT is required | `release/{Release-Name]` |
```

Comment:
@aleoyakas - small thing but make the brackets consistent in the placeholder, i.e. `release/{Release-Name}`